mix.exs

```diff
@@ -25,20 +38,39 @@ defmodule Talon.Mixfile do
   defp elixirc_paths(_),     do: ["lib"]
 
   defp deps do
+    # If using umbrella app, need to always include deps used by siblings
+    test_env = if talon_in_umbrella?(), do: [:dev, :prod, :test], else: :test
+
     [
       {:inflex, "~> 1.7"},
-      {:phoenix, "~> 1.3.0-rc"},
+      {:phoenix, "~> 1.3.0"},
       {:phoenix_html, "~> 2.6"},
       {:phoenix_ecto, "~> 3.2"},
       {:scrivener_ecto, "~> 1.1"},
-      {:postgrex, ">= 0.0.0", only: :test},
       {:phoenix_slime, "~> 0.9"},
       {:slime, "~> 1.0", override: true},
-      {:ecto_talon, github: "talonframework/ecto_talon"},
       {:mix_test_watch, "~> 0.3", only: :dev, runtime: false},
-      {:gettext, "~> 0.11", only: :test},
-      {:dialyxir, "~> 0.5.0", only: [:dev]}
+      {:dialyxir, "~> 0.5", only: [:dev]},
+      {:ecto_talon, github: "talonframework/ecto_talon"},
       # {:ecto_talon, path: "../ecto_talon", only: :test},
+
+      {:postgrex, ">= 0.0.0", only: [:dev, :prod, :test]},
+      {:gettext, "~> 0.11", only: test_env}
     ]
   end
+
+  def talon_in_umbrella? do
+    System.get_env("TALON_IN_UMBRELLA") == "true"
+  end
+
+  # From Mix.Phoenix
+  # This doesn't work when pulling Talon into an umbrella using a symbolic link
+  # TODO: remove
+  def in_umbrella?(app_path) do
+    umbrella = Path.expand(Path.join [app_path, "..", ".."])
+    mix_path = Path.join(umbrella, "mix.exs")
+    apps_path = Path.join(umbrella, "apps")
+    IO.inspect apps_path, label: "apps_path"
```

Comment:
The inspect should be removed before merging.

Reply:
Per the TODO, I'll remove this function altogether.